src/comparisonActions.test.js

```diff
@@ -42,4 +59,83 @@ describe('The comparions actions', () => {
     await createDirectories(mockFs, config).catch(err => console.log(err));
     expect(mockFs.mkdirSync.mock.calls.length).toBe(3);
   });
+
+  it('deletes generated differences from the remote bucket before fetching baseline images', async () => {
```

Comment:
can we make it more explicit on what this is testing?
Its also getting each image